streaming/src/main/scala/org/apache/spark/streaming/DStreamGraph.scala

```diff
@@ -79,7 +79,7 @@ final private[streaming] class DStreamGraph extends Serializable with Logging {
   def remember(duration: Duration) {
     this.synchronized {
       if (rememberDuration != null) {
-        throw new Exception("Remember duration already set as " + batchDuration +
+        throw new Exception("Remember duration already set as " + rememberDuration +
```

Comment:
Sounds good. While you're here though, can you make this use string interpolation, and use `require` (so as to avoid generic `Exception`)?